go.mod

```diff
@@ -14,23 +14,23 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.9.0
 	github.com/temporalio/ui-server/v2 v2.28.0
-	go.temporal.io/api v1.34.0
+	go.temporal.io/api v1.39.0
 	go.temporal.io/sdk v1.27.0
-	go.temporal.io/server v1.24.2
-	google.golang.org/grpc v1.65.0
+	go.temporal.io/server v1.26.0-120.0.20240904002151-40431ec95546
```

Comment:
We won't be able to merge this until it is in a tagged version. If it only lands in a non-stable tagged version (i.e. 1.26 series and not one of the 1.25 patches), we will need to change the branch this is targeting to `next` (once that branch is created).

Reply:
The branch exists (it's called `next-server`).

Reply:
:+1: So once this is in a tagged non-GA version, this can be updated and the target if this PR can be changed to target that branch (or we can close and recreate the PR if needed, unsure if GH supports changing target branch).

Reply:
Updated the base branch to `next-server`.